frontend/src/components/History/History.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import HistoryItem from './HistoryItem'
+
+export default class History extends Component {
```

Comment:
History is too common and may be mislead with browser history. TransactionHistory or TransactionsList or TransactionsTable are more concrete and better describe the role of the component.